server/src/main/java/org/elasticsearch/common/util/concurrent/ThreadContext.java

```diff
@@ -193,33 +193,38 @@ public StoredContext newEmptySystemContext() {
      */
     public StoredContext newTraceContext() {
```

Comment:
Do the javadocs for this method need some tweaking?

Reply:
No, functionally it's exactly the same. It just avoids creating a copy of the immutable current context if that copy will be equal anyways.